iosApp/PhongKMMIC/Sources/Presentation/ViewModifiers/PrimaryButton.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  PrimaryButton.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 20/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct PrimaryButton: ViewModifier {
```

Comment:
What do you think about renaming `PrimaryButton` to `PrimaryButtonModifier`? 
The current name is a bit confuse IMO 🙏

Reply:
I solved it in 92d5442